tools/check-node-version.js

```diff
@@ -9,8 +11,8 @@ var currentVersion = Number(process.version.match(/^v(\d+\.\d+)/)[1]);
 var minVersion = Number(fs.readFileSync(path.join(__dirname, '../', '.nvmrc'), 'utf8'));
 
 if (currentVersion < minVersion) {
-    console.log("You are using Node v" + currentVersion + ".\n" +
-        "Frontend requires Node v" + minVersion + " or later.\n" +
-        "If you're using NVM, you can 'nvm use'...");
+    console.log('You are using Node v' + currentVersion + '.\n' +
+        'Frontend requires Node v' + minVersion + ' or later.\n' +
+        'If you\'re using NVM, you can \'nvm use\'...');
```

Comment:
the annoying thing with template strings is needing to remove the indents in examples like this?